docs/site/Relation-generator.md

```diff
@@ -0,0 +1,106 @@
+---
+lang: en
+title: 'Relation generator'
+keywords: LoopBack 4.0, LoopBack 4
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Relation-generator.html
+summary: summary
+---
+
+# Relation generator
```

Comment:
Please remove this level-1 heading. Our docs-site generator will generate the heading using `title` specified on line 3 above.

Reply:
Done.